### PR TITLE
[DSS-354] Tokens - Add Sage Tokens to Repo

### PR DESCRIPTION
**Pull request by anechol**
# Description

Adds an initial set of Sage tokens to the repo. Further formatting and fine-tuning of names will be done in a future PR. 

[DSS-354](https://kajabi.atlassian.net/browse/DSS-354)

[DSS-354]: https://kajabi.atlassian.net/browse/DSS-354?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/core/src/global/styles/tokens/tokens.json [hunk 1]
```diff
@@ -0,0 +1,437 @@
+{
+  "sage": {
+    "color": {
+      "white": {
+        "white": {
+          "value": "#ffffff",
```
Comment (pixelflips) [inline libs/core/src/global/styles/tokens/tokens.json]: Seems all the other color values are uppercase, should this one be as well?

File: libs/core/src/global/styles/tokens/tokens.json [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+          "type": "color",
+          "description": "Backgrounds, primary/destructive CTA text"
+        }
+      },
+      "black": {
+        "black": {
+          "value": "#000000",
```
Comment (pixelflips) [inline libs/core/src/global/styles/tokens/tokens.json]: No preference really, but should we use the shorthand hex notation where possible?

**Review by pixelflips — APPROVED** [review]
A couple of tiny non-blocking comments, but LGTM! Nice work! 👍🏼

**Review by ju-Skinner — APPROVED** [review]
LGTM...great initial commit to getting us on our way with Tokens.

File: libs/core/src/global/styles/tokens/tokens.json (hunk 1 above)
Comment (anechol) [reply libs/core/src/global/styles/tokens/tokens.json]: Good callout, I will set them all to uppercase.

File: libs/core/src/global/styles/tokens/tokens.json (hunk 2 above)
Comment (anechol) [reply libs/core/src/global/styles/tokens/tokens.json]: I've updated this for now. Previously, I had issues with linting rules, but it doesn't seem to be a problem at the moment.

**Review by goodwinchris — APPROVED** [review]

**Pull request merged**